geode-core/src/distributedTest/java/org/apache/geode/internal/cache/RemoteTransactionDUnitTest.java

```diff
@@ -633,19 +629,19 @@ public void testPRTXGet() {
       @Override
       public Object call() {
         TXManagerImpl mgr = getCache().getTxManager();
-        assertTrue(mgr.isHostedTxInProgress(txId));
+        assertThat(mgr.isHostedTxInProgress(txId)).isTrue();
         TXStateProxy tx = mgr.getHostedTXState(txId);
         System.out.println("TXRS:" + tx.getRegions());
-        assertEquals(3, tx.getRegions().size());// 2 buckets for the two puts we
+        assertThat(3).isEqualTo(tx.getRegions().size());// 2 buckets for the two puts we
```

Comment:
Small nitpick, but this and a few other assertions in this class should be swapped around so that the expected and actual values are in the correct order, e.g.:
```
assertThat(tx.getRegions().size()).isEqualTo(3);
```